cpp/ql/test/library-tests/dataflow/taint-tests/stringstream.cpp

```diff
@@ -32,18 +32,18 @@ void test_stringstream_string(int amount)
 	sink(ss2 << source()); // $ ast,ir
 	sink(ss3 << "123" << source()); // $ ast,ir
 	sink(ss4 << source() << "456"); // $ ast,ir
-	sink(ss5 << t); // $ ast MISSING: ir
+	sink(ss5 << t); // $ ast,ir
 
 	sink(ss1);
 	sink(ss2); // $ ast,ir
 	sink(ss3); // $ ast MISSING: ir
 	sink(ss4); // $ ast,ir
-	sink(ss5); // $ ast MISSING: ir
+	sink(ss5); // $ ast MISSIN,ir
```

Comment:
Typo. I wonder how this comment gets interpreted.

Reply:
Hm, 🤔 Good question! I would think it would be interpreted as `// $ ast,MISSIN,ir` could I could be wrong. Anyway, I'll fix it up.

Reply:
Fixed in dd6b27df24180f7bcd0e40fc1ed996c301b451b5.